docs/elements/layer.md

```diff
@@ -63,6 +63,26 @@ and the fetched `<map-title>` element is used.
 
 ---
 
+### `opacity`
+
+The `opacity` attribute is used to set the opacity of the `<layer->` element. Just
+like the opacity slider on the map, adding this attribute can change the initial
+opacity of any layer.
```

Comment:
```suggestion
The `opacity` attribute is used to set the initial opacity of the `<layer->` element.
When the `opacity` attribute is not present, the opacity is set to "1.0" by default.
```

Reply:
Updated this suggestion to include valid opacity range and mention the layer control opacity input slider control